ui/src/components/BaseFormView/BaseFormView.test.tsx

```diff
@@ -66,3 +73,63 @@ it('should pass default values to custom component correctly', async () => {
 
     expect((customModal as HTMLSelectElement)?.value).toEqual('input_three');
 });
+
+it.each([
+    { page: 'configuration', config: getGlobalConfigMockGroupsForConfigPage(), service: 'account' },
+    { page: 'inputs', config: getGlobalConfigMockGroupsFoInputPage(), service: 'demo_input' },
+])('entities grouping for page works properly %s', async ({ config, page, service }) => {
+    setUnifiedConfig(config);
+    const getElementsByGroup = (group: string) => {
```

Comment:
these three methods can be extracted out from the test case. It would be easier to read the test case itself

Reply:
true, moved on top of file